code/game/objects/items/weapons/holy_weapons.dm

```diff
@@ -145,7 +145,7 @@
 	icon_state = "sord"
 	item_state = "sord"
 	slot_flags = SLOT_BELT
-	force = 2
+	force = 4.13
```

Comment:
I mean, it's PI to 3 significant figures backwards... does... does that mean anything here?